doc/reference/config/project_defaults.rst

```diff
@@ -0,0 +1,50 @@
+project_defaults
+----------------
+
+Specify default values for stanzas ``authors``, ``maintainers``, and ``license``
+of the dune-project file when initializing a project with ``dune init proj``.
+The format of the 'project_defaults' stanza is as follows:
+
+.. code:: dune
+
+   (project_defaults
+    <optional-fields>)
+
+``<optional-fields>`` are:
+
+.. describe:: (authors <string(s)>)
+
+   Specify authors.
```

Comment:
I wonder, if this could link to the definition of `authors` in the `dune-project` reference?

Reply:
I couldn't find any relevant definition of `authors` in the reference, I'm sure I am just glancing over it though. I made all the other requested changes as well as adding a `.. versionadded 3.17` to the top of the doc page!